ios/Classes/Extensions/Flutter/FlutterMethodCall+Extensions.swift

```diff
@@ -82,13 +82,13 @@ extension FlutterMethodCall {
     }
 
     // MARK: - Parameter Values
-
+    
     func intValue(for key: String, defaultValue: Int32 = 0, result: FlutterResult) -> Int32 {
         guard let parameters = parameters else {
             return defaultValue
         }
 
-        if !contains(keys: [key]) {
+        if !contains(key: key) {
```

Comment:
@phranck was this change intentional? Looks somehow strange, as nothing else was renamed.

Reply:
Yes, it has been changed by intention. A request of looking for an array of keys is never used but looking for one key.